lib/azure/armrest/model/base_model.rb

```diff
@@ -22,6 +22,8 @@ def self.attr_hash(*attrs)
 
       attr_hash :tags
 
+      attr_accessor :response_headers
```

Comment:
We don't want to add headers for every object, because they would all be identical, and would create unnecessary memory bloat. It's meant to only be set once per request.